index.bs

```diff
@@ -1943,53 +1947,88 @@ Constructors</h4>
 
-                1. Set the {{BaseAudioContext/state}} attribute of the {{AudioContext}} to "{{AudioContextState/running}}".
+                3. Set the {{[[rendering thread state]]}} to <code>running</code> on the {{AudioContext}}.
 
                 1. <a href="https://html.spec.whatwg.org/multipage/media.html#queue-a-media-element-task">
-                    queue a media element task</a> to <a spec="dom" lt="fire an event">fire an event
-                    </a> named {{BaseAudioContext/statechange}} at the {{AudioContext}}.
-        </del>
-        <ins cite=#2400>
-            1. Attempt to <a href="#acquiring">acquire system resources</a> to use a
-                following audio output device based on {{AudioContext/[[sink ID]]}} for
-                rendering:
+                    queue a media element task</a> to execute the following steps:
 
-                * The default audio output device for the empty string.
-                * A audio output device identified by {{AudioContext/[[sink ID]]}}.
-            
-                In case of failure, abort the following steps.
+                    1. Set the {{BaseAudioContext/state}} attribute of the {{AudioContext}} to "{{AudioContextState/running}}".
 
-            1. Set the {{[[rendering thread state]]}} to <code>running</code> on the
-                {{AudioContext}}.
+                    1. <a href="https://html.spec.whatwg.org/multipage/media.html#queue-a-media-element-task">
+                        queue a media element task</a> to <a spec="dom" lt="fire an event">fire an event
+                        </a> named {{BaseAudioContext/statechange}} at the {{AudioContext}}.
+            </del>
+            <ins cite="#2400 #2567">
+                1. Attempt to <a href="#acquiring">acquire system resources</a> to use a
+                    following audio output device based on {{AudioContext/[[sink ID]]}} for
+                    rendering:
 
-            1. <a href="https://html.spec.whatwg.org/multipage/media.html#queue-a-media-element-task">
-                Queue a media element task</a> to execute the following steps:
+                    * The default audio output device for the empty string.
+                    * A audio output device identified by {{AudioContext/[[sink ID]]}}.
+                
+                    Upon failure of resource acquisition,
+                        <a href="https://html.spec.whatwg.org/multipage/media.html#queue-a-media-element-task">
+                        queue a media element task</a> to
+                        <a spec="dom" lt="fire an event">fire an event</a> named
+                        {{AudioContext/error}} at the {{AudioContext}}, and abort
+                        the following steps.
 
-                1. Set the {{BaseAudioContext/state}} attribute of the {{AudioContext}}
-                    to "{{AudioContextState/running}}".
+                1. Set the {{[[rendering thread state]]}} to <code>running</code> on the
```

Comment:
Shouldn't this say (same with the others... internal slots are for instances, but this never talks about what "this" is): 

```suggestion
                1. Set [=this=] {{[[rendering thread state]]}} to <code>running</code> on the
```

Reply:
Thanks for the suggestion! Done.